### PR TITLE
Feature/fix alignment for cards

### PR DESCRIPTION
**Pull request by HarisShahidHameed**
Trello-link : https://trello.com/c/Xeoj5rHX/56-show-message-no-data-available-on-dashboard

### CODE REVIEW

File: src/containers/Dashboard/Quizzes/Quiz.js [hunk 1]
```diff
@@ -27,21 +27,30 @@ class Quizzes extends Component {
...
 
   render() {
-    const quizCards = this.state.quizzes.map((quiz, index) => (
-      <div className="interview-prep-quiz" key={index}>
-      <div className="quiz-card shadow-lg ml-5 mr-4 mb-5">
-        <QuizCard heading={quiz.name} value={quiz.quizId}/>
+    let cards;
+    if(this.state.quizzes.length > 0) {
+      cards = this.state.quizzes.map((quiz, index) => (
+        <div className="interview-prep-quiz" key={index}>
```
Comment (abdulhaqshah) [inline src/containers/Dashboard/Quizzes/Quiz.js:33]: code is not indent

File: src/containers/Dashboard/Tutorials/Tutorial.js [hunk 2]
```diff
@@ -27,20 +27,31 @@ class Card extends Component {
   }
   
   render() {
+    let cards;
+    if(this.state.tutorials.length > 0) {
+      cards = this.state.tutorials.map((tutorial, index) => (
+        <div className="interview-prep-tutorial" key={index}>
```
Comment (abdulhaqshah) [inline src/containers/Dashboard/Tutorials/Tutorial.js:33]: not indent

File: src/containers/Dashboard/Tutorials/Tutorial.js [hunk 3]
```diff
@@ -27,20 +27,31 @@ class Card extends Component {
...
+      )
+    }
+    
     return (
       <Fragment>
         <div>
           <h5 className="headings ml-5 mb-5 mt-5">Tutorials</h5>
         </div>
-        <div className="row">
+        <div className="d-flex justify-content-center row">
```
Comment (abdulhaqshah) [inline src/containers/Dashboard/Tutorials/Tutorial.js:52]: class name??

**Review by abdulhaqshah — CHANGES_REQUESTED** [review]

File: src/containers/Dashboard/Quizzes/Quiz.js (hunk 1 above)
Comment (HarisShahidHameed) [reply src/containers/Dashboard/Quizzes/Quiz.js:33]: Done.

File: src/containers/Dashboard/Tutorials/Tutorial.js (hunk 2 above)
Comment (HarisShahidHameed) [reply src/containers/Dashboard/Tutorials/Tutorial.js:33]: Done.

File: src/containers/Dashboard/Tutorials/Tutorial.js (hunk 3 above)
Comment (HarisShahidHameed) [reply src/containers/Dashboard/Tutorials/Tutorial.js:52]: Done.

**Pull request merged**